components/icons/nowcasting-logo.tsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image';
+
+type NowcastingLogoProps = {
+  width: string;
+  height: string;
+};
+
+export const NowcastingLogo: React.FC<NowcastingLogoProps> = ({
+  width,
+  height,
+}) => (
+  <a href="https://www.openclimatefix.org/" target="_blank" rel="noreferrer">
```

Comment:
i'm not sure if this is necessary, but i see a lot of sites use <Link> instead of <a>, so maybe that might be the move